docs/fixmystreet.md

```diff
@@ -57,6 +64,35 @@ You could store a copy of our configuration file in a secure location and have
 a script copy this into the right place and ensure the permissions were correct
 when you launch an instance.
 
+## Adding your own cobrand
+
+If you want to build a custom AMI with [your own cobrand](https://fixmystreet.org/customising/)
+baked in, we've provided a variation on our standard builder to cater for this.
+
+First, create a directory called under `files/` called `cobrand` and place your
+cobrand files into it.
+
+For example, if you maintain your cobrand in its own repository you could clone
+a working copy directly into this directory.
```

Comment:
Yeah, I was trying to get at that by saying "directly into this directory" (I had someone running `git clone <foo> .` within that directory in mind) but I guess it could be clearer.